src/Redux/Actions/Questionnaires/index.js

```diff
@@ -63,6 +73,16 @@ export const createQuestionnaire = data => (dispatch) => {
       dispatch(rejectedCreateQuestionnaire());
     });
 };
+
+export const deleteQuestionnaire = id => (dispatch) => {
+  dispatch(startDeleteQuestionnaire());
+  Axios.delete(`/my-questionnaires/${id}`)
+    .then(() => {
+      dispatch(questionnaireDeleted());
+    })
+    .catch(() => {
+      dispatch(rejectedDeleteQuestionnaire());
+
```

Comment:
You forgot to close your catch (and your function), the code won't compile because of it.

Reply:
I solved conflicts and must have deleted it by mistake - but it is fixed now!